setup.py

```diff
@@ -62,7 +62,7 @@ def run(self):
         'Topic :: Software Development :: Libraries :: Python Modules',
     ],
     packages=['babel', 'babel.messages', 'babel.localtime'],
-    package_data={'babel': ['global.dat', 'localedata/*.dat']},
+    package_data={'babel': ['global.dat', 'locale-data/*.dat']},
```

Comment:
I filed https://github.com/python-babel/babel/issues/260 because this is another change and should be done in a nown commit.